install.sh

```diff
@@ -208,9 +198,14 @@ echo crew >> .git/info/sparse-checkout
 echo tools   >> .git/info/sparse-checkout
 git fetch origin "${BRANCH}"
 git reset --hard origin/"${BRANCH}"
+echo -e "${YELLOW}Updating crew package information.${RESET}"
 crew update
 
-yes | crew install $LATE_PACKAGES 
+crew_installed_packages=$(jq  '.installed_packages[] .name' "${CREW_CONFIG_PATH}/device.json" | sed -e 's/"//g'   | sort)
+crew_core_packages=$(cat "${CREW_LIB_PATH}/tools/core_packages.txt" | sort)
```

Comment:
If we do this and core packages are updated, we need to be watchful to update core_packages.txt as needed.

Reply:
Yes. We should update logic in crew's `crew update` for this.

Reply:
Opened https://github.com/skycocker/chromebrew/issues/5865 to track this.